noxfile.py

```diff
@@ -47,6 +47,7 @@
     "blacken",
     "mypy",
     "pytype",
+    # "mypy_samples",  # TODO: uncomment when the checks pass
```

Comment:
Is this intended to stay commented out?

If so, maybe link to https://github.com/googleapis/python-pubsub/issues/536

Reply:
Edit: Just saw the comment above. I'm okay leaving this as-is, assuming it's very temporary.

Reply:
Yes, it's very temporary indeed. I wanted to keep the refactoring changes from #536 separate for easier rollback, should that be needed (so that we wouldn't be forced to purge other things as well).